src/types/ProposalFieldValue.ts

```diff
@@ -1,21 +1,28 @@
 import type { JSONSchemaType } from 'ajv';
 import type { ApplicationFormField } from './ApplicationFormField';
+import type { Writable } from './Writable';
 
-export interface ProposalFieldValue {
-	id: number;
+interface ProposalFieldValue {
+	readonly id: number;
 	proposalVersionId: number;
 	applicationFormFieldId: number;
 	position: number;
 	value: string;
-	createdAt: Date;
-	applicationFormField?: ApplicationFormField;
+	readonly createdAt: Date;
+	readonly applicationFormField?: ApplicationFormField;
+	readonly isValid: boolean;
 }
 
-export type ProposalFieldValueWrite = Omit<
+type ProposalFieldValueWrite = Omit<
```

Comment:
Ah I missed this -- now that you have `WritableProposalFieldValue` you should take this type out (basically, this is the way we *used* to define writable versions of types, but now we have the `Writable` class).

This might require changes to other places that use it.

Alternatively, if you don't want to do that refactor in here you should take out the `WritableProposalFieldValue` definition and just use `ProposalFieldWrite` there instead (and add `isValid` to its list of Omitted fields).

Reply:
This seems like a reasonably large refactor (as I would have to change the proposal version schema as well) so i think I'll drop the WritableProposalFieldValue for now and would happily do a larger refactor as a separate PR!

Reply:
That sounds right -- and you don't need to do the refactor; it's on my list to continue the refactor I've been doing for all the other endpoints / schemas as well!